platform/android/MapLibrePlugin/local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Mon Nov 25 21:55:41 CET 2024
+sdk.dir=/Users/bart/Library/Android/sdk
```

Comment:
This should be a local file. No warning/error generated when building

Reply:
You're right!